tests/ui/derive-arbitrary/enum/expected

```diff
@@ -0,0 +1,24 @@
+Checking harness check_comparison...
+SUCCESS\
+"assertion failed: disc >= -1 && disc <= 1"
+
+SUCCESS\
+"assertion failed: disc == 1"
+
+SUCCESS\
+"assertion failed: disc == -1"
+
+SUCCESS\
+"assertion failed: disc == 0"
+
+
+Checking harness check_enum_wrapper...
+
+SUCCESS\
```

Comment:
Are you relying on this being reported as "UNREACHABLE" if the variant is not possible? If so, you might want to use the brand new `cover` macro instead :)

Reply:
Yes. I was thinking about that while writing all these tests. =)